composer.lock

```diff
@@ -575,16 +575,16 @@
         },
         {
             "name": "swaggest/php-code-builder",
-            "version": "v0.2.41",
+            "version": "v0.2.42",
```

Comment:
There's a bunch of unrelated changes in the composer.lock file. Let's keep it as it was.

Reply:
I'll rollback the whole file. Yet I fail to see how I have cause this? I did run the code suggested in the README file. Why would it not adhere to the specs in the `composer.json` file?